src/components/breadcrumbs/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Crumb } from './types';
+import { getMetaInfoForTopic } from '../../service/get-meta-info-for-topic';
+
+const categoryToCrumb: { [key: string]: Crumb } = {
+    products: { text: 'Products', url: '/developer/products' },
```

Comment:
Not 100 percent sure about this one but I thought we can use Amanda's utility function `getURLPath` and not reference developer in the URLs?

Reply:
Yeah, I think for now that can be used here to avoid referencing /developer.